docs/mining/proof-of-stake.md

```diff
@@ -2,29 +2,142 @@
+Using a stake pool is beneficial because the servers are geographically
+distributed and redundant which vastly increases the odds that your vote
+will always be cast even if a network or server outage occurs.
+The stake pools charge a fee to pay for server costs and system
+administration labor.
+
+---
+
+## **<i class="fa fa-life-ring"></i> Sign up for a stake pool**
+
+These stake pools are officially recognized:
+
+* [<i class="fa fa-external-link-square"></i> https://dcr.stakepool.net](https://dcr.stakepool.net)
+* [<i class="fa fa-external-link-square"></i> https://dcr.stakeminer.com](https://dcr.stakeminer.com)
+* [<i class="fa fa-external-link-square"></i> http://pool.d3c.red](http://pool.d3c.red)
+* [<i class="fa fa-external-link-square"></i> https://dcrstakes.com](https://dcrstakes.com)
+* [<i class="fa fa-external-link-square"></i> https://stakepool.dcrstats.com](https://stakepool.dcrstats.com)
+* [<i class="fa fa-external-link-square"></i> https://stake.decredbrasil.com](https://stake.decredbrasil.com)
+* [<i class="fa fa-external-link-square"></i> http://stakepool.eu](http://stakepool.eu)
+* [<i class="fa fa-external-link-square"></i> http://dcrstakepool.getjumbucks.com](http://dcrstakepool.getjumbucks.com)
+
+You can find a comparison of each pool's fees and statistics by visiting the
+[<i class="fa fa-external-link-square"></i> Decred website](https://decred.org)
+and clicking the 'Find a stake pool' link at the top right.
+
+Each pool runs the same software which will walk you through the ticket buying
+process. The instructions are also below so you may view them without signing
+up for a pool.
 
 ---
 
-## **<i class="fa fa-ticket"></i> Purchasing tickets**
+## **<i class="fa fa-ticket"></i> Automatic Purchasing Of Tickets**
+
+[<i class="fa fa-github"></i> dcrticketbuyer](https://github.com/decred/dcrticketbuyer)
+is a highly configurable utility for purchasing tickets.  The GitHub project
+page
+[<i class="fa fa-external-link-square"></i>https://github.com/decred/dcrticketbuyer](https://github.com/decred/dcrticketbuyer)
+provides an overiew of the options and how to run it.
 
-To participate in PoS mining, you will first need to have followed the [Getting Started](/getting-started/overview.md) guide, [Obtain DCR](/getting-started/obtaining-dcr.md) guide, and have unlocked your wallet. 
+<i class="fa fa-exclamation-triangle"></i> ** It is highly recommended
+to try dcrticketbuyer on testnet to make sure it's correctly configured and
+behaves as expected before attempting to use it on mainnet. Once tickets are
+purchased, the funds can be locked for months without any way to retrieve
+them. **
+
+> dcrticketbuyer (Solo Mining)
+
+The bare minimum configuration is to provide the RPC access credentials for
```

Comment:
Even in the case of solo mining, we should still probably tell the user to delegate their voting to a hot voting wallet address with ticketaddress, or describe how to export their voting address' private key from a separate account so that they can vote with a hot wallet.